models/Comment.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+
+const CommentSchema = new mongoose.Schema({
+        
+    comment:{
+        type: String,
+        required: true
+    },
+   
+    postBy: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'Users'
+    }
+})
+
+const Comments = mongoose.model('Comments' , CommentSchema)
```

Comment:
It is common practice to use singular names for Models.
So, `Comment` instead of `Comments`